### ISSUE TITLE
There should be more than one ServiceHost for an instance

### ISSUE THREAD

**Issue by Cyberboss** [tgstation/tgstation-server#398]
This is a separate issue to #320 

We've apparently been using WCF wrong the whole time. Only after #395 is merged do we have the, correct, option of having multiple ServiceHosts per instance all linked to the same ServiceAuthorizationManager

**Comment by Cyberboss** [comment]
One this note, Server should contain a list of IInstances, not fucking servicehosts

**Comment by Cyberboss** [comment]
Won't fix in V3
